Lottery-Game-Chainlink-VRF.md

```diff
@@ -226,9 +226,9 @@ contract RandomWinnerGame is VRFConsumerBase, Ownable {
 The constructor takes in the following params:
 
 - `vrfCoordinator` which is the address of the VRFCoordinator contract
-- `linkToken` is the address of the link token which is the token in which the chainlink takes its payment
+- `linkToken` is the address of the link token which is the token in which the chainlink takes it's payment
```

Comment:
This should be `its`. You can find more [here](https://www.grammarly.com/blog/its-vs-its/).